pkg/apis/acid.zalan.do/v1/crds.go

```diff
@@ -649,12 +649,21 @@ var PostgresCRDResourceValidation = apiextv1.CustomResourceValidation{
 						Type: "integer",
 					},
 					"standby": {
-						Type:     "object",
-						Required: []string{"s3_wal_path"},
+						Type: "object",
 						Properties: map[string]apiextv1.JSONSchemaProps{
+							"cluster": {
+								Type: "string",
+							},
+							"uid": {
+								Type:   "string",
+								Format: "uuid",
+							},
```

Comment:
this can go, I think. Maybe a left over from an earlier version of this PR

Reply:
Cleaned these up too.